libs/components/forms/src/lib/modules/character-counter/character-counter.component.spec.ts

```diff
@@ -44,7 +44,8 @@ describe('Character Counter component', () => {
       fixture.detectChanges();
 
       characterCountComponent =
-        component.inputDirective.skyCharacterCounterIndicator;
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I'd prefer to see the non-null assertion warnings in the console/IDE so they can be addressed at a future date.

Reply:
Ok.... I've been doing these in a lot of places in other stories....

Reply:
Removed